common/djangoapps/enrollment/views.py

```diff
@@ -264,9 +267,12 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
 
                If honor mode is not supported for the course, the request fails and returns the available modes.
 
-               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified"
-               or "professional". If the mode is not supported for the course, the request will fail and return the
-               available modes.
+               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified",
+               "professional" or "credit". If the mode is not supported for the course, the request will fail and
+               return the available modes.
+
+               The enrollment attributes are also set against the course mode if provided 'enrollment_attributes' are
```

Comment:
@mhoeber The basic idea is that for any enrollment, the caller can specify a set of arbitrary key/value pairs that we call "enrollment attributes".  One example is for credit courses, where we specify the credit provider ID (e.g. "hogwarts") for enrollments in a "credit" mode.